dist/win/build.ps1

```diff
@@ -62,25 +70,25 @@ if ($clean -and (Test-Path -Path $appPath)) {
 	--module-path ../../target/mods `
 	--module org.cryptomator.desktop/org.cryptomator.launcher.Cryptomator `
```

Comment:
just found this in the macOS build script review.

the module name (and potentially the main class name) need to be configurable. I would suggest a single variable for the complete string.

Reply:
@tobihagemann @infeo please agree on a common standard for both build scripts

Reply:
fixed in [75093f1](https://github.com/cryptomator/cryptomator/pull/2242/commits/75093f1c09f322aab752bdb6bb59e089565042d6)